app/services/idv/session.rb

```diff
@@ -66,21 +65,18 @@ def create_profile_from_applicant_with_password(user_password)
 
       profile.activate unless profile.reason_not_to_activate
 
-      self.pii = profile_maker.pii_attributes
       self.profile_id = profile.id
       self.personal_key = profile.personal_key
 
-      cache_encrypted_pii(user_password)
```

Comment:
This wrote the PII to the Idv session as `decrypted_pii`. At this point the PII was available as `decrypted_pii` and `pii` in the IdV session. Later in the old implementation of `move_pii_to_user_session` the `decrypted_pii` value was deleted from the IdV session and copied over into the user session.

I cannot explain any of this but I'm reasonably confident none of it is necessary.